src/components/articles/Article.js

```diff
@@ -70,14 +76,30 @@ const Article = ({
                     )}
                 />
               </Col>
-              <Col className="article-liking" span={4}>Liking</Col>
-              <Col className="article-liking" push={14} span={4}>
+              <Col className="article-liking" push={8}>
                 {token ? (
                   <Rate className="rating" onChange={rateArticle} defaultValue={localStorage.getItem('rating')} />
                 ) : (
                   <Rate className="rating" onChange={rateArticle} value={0} />
                 )}
               </Col>
+              <Col push={14}>
+                <Popover
+                  placement="right"
+                  content={
+                      (
+                        <div>
+                          <Facebook url={url + article.slug} shareText={article.title} />
```

Comment:
Is there a way you can add `{url + article.slug}` to a variable then just call it here?

Reply:
I tried doing that but then I was not able to access the article since there is no return which means that if I have to define a variable I have to do it out of the `const Article`.